docker/README.md

```diff
@@ -1,43 +1,95 @@
-More advanced command line options can be given afterwards, like this:
+This will start the ASReview LAB server on port 5000 with default command line
+options and make it accessible to the host at http://localhost:5000
 
- ```
- docker run -p 9000:9000 asreview/asreview --port 9000
- ```
+A special command line interface image is available as `asreview/asreview-cli`.
+Make sure to use the -ti flag to start the image in interactive mode.
+
+
+## Advanced usage
+
+Docker is a very powerful tool for running software in a container, and many
+different options are available to customize the behavior of the container.
+
+To create a container for ASReview LAB and name the container, you can use the
+following command:
+
+```
+docker create --name asreview-lab -p 5000:5000 asreview/asreview
+```
+
+To run this container, use the following command:
+
+```
+docker start asreview-lab
+```
 
-## Running simulations from Docker
+To use a specific image of ASReview LAB, in this case `asreview:0.19`, you can
+use the following command. For a list of available versions, visit the
+[DockerHub page](https://hub.docker.com/r/asreview/asreview/tags/).
 
-ASReview has as well a Docker image for the command line interface. It contains as well the other extensions: visualization, statistics, hyperopt, covid19
+```
+docker create --name asreview-lab -p 5000:5000 asreview/asreview:0.19
+```
 
-The entrypoint of this container is "asreview", so other parameters need to be appended.
+To run multiple containers of asreview LAB at the same time, a new port has to be
+specified for each container. Adjust the host port to the port you want to use.
+```
+docker create --name asreview-0.19 -p 5019:5000 asreview/asreview:0.19
+docker create --name asreview-0.17.1 -p 5017:5000 asreview/asreview:release-0.17.1
+```
 
+Advanced command line options can be given after the image, like this:
 ```
-docker run -ti asreview/asreview-cli -h
+docker create --name asreview-lab -p 5000:5000 asreview/asreview --seed 12345
 ```
 
-Then inside the container all normal asreview CLI commands can be executed. This requires very likely to mount a path from the host into Docker
-in order to exchange data files between host and container. This gets done by adding the "-v" parameter accordingly to the docker run command.
+### Connect to the local project folder
+
+To attach the container to your local project folder (or any other local
+folder), the -v flag can be used in the following way. Adjust
+`path-to-your-folder` to your local folder. When a project folder is specified,
+ASReview LAB will store and load all its projects from this folder.
+Note that multiple containers can access the same folder, making switching
+between versions even easier than before.
 
+```
+docker create --name asreview-lab -p 5000:5000 -v path-to-your-folder:/project_folder asreview/asreview
```

Comment:
Interesting solution. I'm wondering why `/.asreview` doesn't work here, but I assume it doesn't :)

Reply:
I tried, but it did not. I think I'll have to take a look inside a container to see what folder it does store too, but that will take a while.

Reply:
Nice solution for now :)